_docs/_latest/develop/dev/gcp-api.md

```diff
@@ -0,0 +1,220 @@
+---
+title: GCP API
```

Comment:
Can we call this "Forseti API"? We shouldn't use GCP like that, so we'd need to use the full product name otherwise.

Reply:
If we use `Forseti API`, it would refer to the API that forseti exposes for other people to interact with forseti functionalites, which that is not the case here.  So, it really is better as GCP API.  So, thank you for adding the full product name in the first sentence below.

Reply:
Changing to the full product name as discussed, we can revert if it turns out we're okay on branding (awaiting more info here).

Reply:
Of course I almost immediately got confirmation back, so I've reverted - we're good to go here :)